docs/release-notes/8.2.0-1.md

```diff
@@ -0,0 +1,66 @@
+# Percona Server for MySQL 8.2.0-1 (2024-01-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.2.0-1] includes all the features and bug fixes available in the
+MySQL 8.2 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.2 code base. In this release Percona does not introduce any new features, improvements or bug fixes.
```

Comment:
We did several of visible changes as part of the rebase/merge process
See https://perconadev.atlassian.net/browse/PS-8979 for more details
* Removed remainders of Percona-Specific Encryption features (old 5.7 encrypted bilnog compatibility)
* Removed deprecated "Extended mysqlbinlog" (https://docs.percona.com/percona-server/8.0/extended-mysqlbinlog.html)
* Removed deprecated "rocksdb_strict_collation_check" and "rocksdb_strict_collation_exceptions" RocksDB system variables (https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_strict_collation_check)